### ISSUE TITLE
Reproduction results on the BraTS2020 dataset

### ISSUE THREAD

**Issue by Seauagain** [VinyehShaw/HFF#2]
Hi, dear authors. I sincerely appreciate that you open-sourced HFF-Net. I believe the framework you proposed is truly spectacular, and has made a great contribution to the biomedical image segmentation community.

I have been trying to reproduce your experimental results on the [BraTS2020 dataset](https://www.kaggle.com/datasets/awsaf49/brats20-dataset-training-validation). I carefully read the README documentation and strictly followed the steps outlined: downloaded the dataset and trained for 450 epochs using the default configuration. However, the validation set Dice scores I obtained are: **ET ~80%, TC ~85%, WT ~91%**. There is still a certain gap compared to the results mentioned in the paper (**especially ET's 87%**).  

I have checked my implementation process but still cannot achieve the performance reported in the paper. Therefore, I would like to ask if there are any key details or tricks that I might have overlooked? 
Any suggestions would be greatly helpful to me. Thank you!

My training logs read as follows:
```
===============================================================
| Epoch 450/450                                               |
---------------------------------------------------------------
| Train Sup Loss LF: 0.1099    | Train SUP Loss HF: 0.1474    |
| Train Unsup Loss: 0.0290     | Train Total Loss: 1.5805     |
---------------------------------------------------------------
---------------------------------------------------------------
| Val Sup Loss LF: 0.1908      | Val Sup Loss HF: 0.2322      |
---------------------------------------------------------------
| Val ET Dice LF: 0.7929       | Val ET Dice HF: 0.7079       |
| Val ET HD95 LF: nan          | Val ET HD95 HF: nan          |
| Val TC Dice LF: 0.8542       | Val TC Dice HF: 0.8385       |
| Val TC HD95 LF: 6.7022       | Val TC HD95 HF: 6.3739       |
| Val WT Dice LF: 0.9097       | Val WT Dice HF: 0.8950       |
| Val WT HD95 LF: 6.2468       | Val WT HD95 HF: 6.4924       |
---------------------------------------------------------------
```
